clinica/cmdline.py

```diff
@@ -74,8 +83,15 @@ def setup_logging(verbose: bool = False) -> None:
 @click.group(context_settings=CONTEXT_SETTINGS)
 @click.version_option(version=clinica.__version__)
 @click.option("-v", "--verbose", is_flag=True, help="Increase logging verbosity.")
-def cli(verbose: bool) -> None:
-    setup_logging(verbose=verbose)
+@click.option(
+    "-lfp",
+    "--logging_file_path",
+    type=str,
+    help="The path to the logging file path.",
+    default="/pypeline.log",
```

Comment:
I am not sure I understand the default value here. Unless I am missing something, this will cause permission issues for most users running Clinica without root privileges

Reply:
We should probably not change the current behaviour, which is to write the log file in the cwd. Sure, it's not always guaranteed to be writeable (as reported in the container's case) but `/pypeline.log` definitely will not for local usage.

Other alternatives that I know of are either to write somewhere in `TMPDIR`, or in a so-called "home" directory under something like `~/.clinica` or [`$XDG_STATE_HOME`](https://specifications.freedesktop.org/basedir-spec/0.8/ar01s03.html) preferably. The former provides better availability guarantees but is less discoverable by users. That latter provides better flexibility but would require more work to manage the appropriate paths.

Reply:
Another alternative is also to introduce an override via an environment variable, like `CLINICA_LOG_DIR`.